swagger_server/dal/neo4j.py

```diff
@@ -0,0 +1,96 @@
+from typing import List, Dict
+
+from swagger_server.helpers import db
+
+
+class Neo4jProperty:
```

Comment:
I am not sure I understand the responsibilities of these classes defined here. We don't want to implement an ORM of our own. Please these classes are not intended for any future use please remove them.

Reply:
removed